Sample/IdentityServer/Views/Account/Login.cshtml

```diff
@@ -30,6 +30,15 @@
                                     <label asp-for="Password"></label>
                                     <input type="password" class="form-control" placeholder="Password" asp-for="Password" autocomplete="off">
                                 </div>
+                                <div class="form-group">
```

Comment:
I think a wiki article saying this is optional would be better. I mean, I want to have the code with the basic and then this kind of multiple domain as an extra, not as a default.

Don't get me wrong, I don't dislike the idea. I will approve, but probably comment this part out.